platform/viewer/src/studylist/StudyListWithData.js

```diff
@@ -51,9 +55,10 @@ class StudyListWithData extends Component {
   };
 
   componentDidMount() {
+    const { appConfig = {} } = this.context;
```

Comment:
Can we make `this.context` equal to the `appConfig` instead of `this.context.appConfig`?
That way we don't need to destructure and provide a default in each of these instances.

When we first set it, just make `context` equal to `appConfig || {}`

Reply:
We could but, AppContext could have more than appConfig prop, plus, we could/should better organize appConfig/extension/or anything else when initially setting AppContext Provider. This is the first step for this improvement and at end we wont pass unnecessary properties into appConfig or AppContext

Reply:
plus I didnt do the whole change because it will involve changing something not covered on this scope (i.e extensions/servers...)

Reply:
That works for me. Thanks for providing good and thoughtful comments ^_^